srcpkgs/gnome-screensaver/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'gnome-screensaver'
+pkgname=gnome-screensaver
+version=3.6.1
+revision=1
+build_style=gnu-configure
+configure_args="--enable-compile-warnings=minimum"
+hostmakedepends="pkg-config automake glib-devel"
+makedepends="gnome-desktop-devel dbus-glib-devel pam-devel intltool"
```

Comment:
intltool should be in hostmakedepends

Reply:
Will fix this asap.

Reply:
once that change is done check if lines 17 to 19 are still of any use

Reply:
@maxice8 Ok. Will do that. Thanks.